spec/rubocop/cop/style/multiline_memoization_spec.rb

```diff
@@ -121,6 +121,18 @@
                                 baz
                               end
                           RUBY
+
+          it_behaves_like 'code with offense',
+                          <<-RUBY.strip_indent,
+                            foo ||= (bar ||
+                                     baz)
+                          RUBY
+                          <<-RUBY.strip_indent
+                             foo ||= begin
+                                       bar ||
+                                      baz
+                                     end
```

Comment:
The identation here seems incorrect. I guess the assumption is that another cop is supposed to fix it, right?

Reply:
Yeah, this was the best I could get it without having to check the indentation of each line.

Reply:
Got it.